.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+
+  # Maintain dependencies for GitHub Actions
```

Comment:
Do we need to specify a package-ecosystem for maven? (Is there one for maven?)

Reply:
Yes we can add maven package manager here.Its supported.
https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem

But there are some pre-conditions : 
  
<img width="717" alt="image" src="https://user-images.githubusercontent.com/102190347/210615352-cf9b8197-6258-4a11-a8aa-d29407672099.png">

We need to research this a bit before adding maven package manager inside updates.